package.json

```diff
@@ -9,6 +9,7 @@
     "@storybook/addon-knobs": "~3.4.12",
     "@storybook/react": "~3.4.12",
     "@storybook/storybook-deployer": "^2.0.0",
+    "@theforeman/vendor-dev": "^0.1.0-alpha.6",
```

Comment:
Do we want to merge dependency on `alpha` version, or do we expect to get it at least beta as part of reviewing this? :slightly_smiling_face:

Reply:
Great question @ezr-ondrej :+1: 

IMO we should merge this PR with version `0.1.0` once it's ready and I got a strong feeling `0.1.0-alpha.6` will end up as `0.1.0`.

Do you have a different suggestion?

Reply:
does it really matter? IMHO we should merge when we think its ready, so we can find out the real issues afterwards? :)

Reply:
> does it really matter? IMHO we should merge when we think its ready, so we can find out the real issues afterwards? :)

OK, agree :+1: